packages/plugin-core/src/constants.ts

```diff
@@ -840,6 +840,17 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
     },
     when: "dendron:pluginActive",
   },
+  TOGGLE_PREVIEW: {
+    key: "dendron.togglePreview",
+    title: `${CMD_PREFIX} Toggle Preview`,
+    icon: `$(open-preview)`,
+    // keybindings: {
```

Comment:
can you uncommen the keyboard bindings here and remove the ones for show preview?